components/centraldashboard/public/components/main-page.js

```diff
@@ -66,7 +68,7 @@ export class MainPage extends PolymerElement {
                         href: '/docs',
                     },
                     {
-                        iframeUrl: '/jupyter/',
+                        iframeUrl: 'http://localhost:8000/test.html',
```

Comment:
😕 ❓

Reply:
I added this so you could easily see how it behaves with a sample iframed page. See the latest update.